### PR TITLE
Made almost every cli tool take a soa_dir

### PR DESCRIPTION
**Pull request by mjksmith**

### CODE REVIEW

**Comment by nhandler** [comment]
Cool, Ship It!

We have a mix of `-d` and `-y` for the short argument (due to collisions with other arguments). Think it is worth trying to make this more uniform?

File: paasta_tools/cli/cmds/metastatus.py [hunk 1]
```diff
@@ -51,6 +52,13 @@ def add_subparser(subparsers):
         '-c', '--clusters',
         help=clusters_help,
     ).completer = lazy_choices_completer(list_clusters)
+    status_parser.add_argument(
+        '-d', '--soa-dir',
+        dest="soa_dir",
+        metavar="SOA_DIR",
```
Comment (solarkennedy) [inline paasta_tools/cli/cmds/metastatus.py]: why do you need the metavar on all of these?

**Comment by solarkennedy** [comment]
I agree shipit and I agree the consistency thing. We have grown up now and I think we are ready to get rid of the "-y" and stick with "-d". (my terrible grep doesn't reveal much conflict?)

But I wouldn't block a shipit with the consistency thing, that can be done in a second pass if you want.

File: paasta_tools/cli/cmds/metastatus.py (hunk 1 above)
Comment (mjksmith) [inline paasta_tools/cli/cmds/metastatus.py]: @solarkennedy I copied it from another cli command.

**Pull request merged**